oceanspy/utils.py

```diff
@@ -23,11 +23,11 @@
     import numba
 
     has_numba = True
-except ImportError:
+except ImportError:  # pragma: no cover
     has_numba = False
 
 
-def compilable(f):
+def compilable(f):  # pragma: no cover
```

Comment:
I think we can also get rid of this because hardly anything in oceanspy is numba compilable any way

Reply:
removed